installer.cmd

```diff
@@ -0,0 +1,253 @@
+::@Kimbyeongjang - (김병장#7165)
+title A32NX Builder
+@echo off

```

Comment:
We should put `@echo off` before the title command, otherwise the command is printed to the cmd window.